validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+
+| Step Number | Action | Data | Expected Result                              |
+|------|--------|--------|-----------------------------------------------|
+| 1 | Install Rancher monitoring stack from Apps catalog | | Monitoring deployment completes               |
+| 2 | Verify Prometheus and Grafana pods are running | | All monitoring components operational         |
+| 3 | Access Longhorn Settings and enable "Longhorn System Monitoring" | | Monitoring integration enabled                |
+| 4 | Access Grafana through Rancher monitoring section | | Grafana dashboard opens                       |
+| 5 | Search for "Longhorn" dashboards in Grafana | | Longhorn dashboards appear in search results  |
+| 6 | Open "Longhorn - Volume" dashboard | | Volume metrics dashboard displays             |
+| 7 | Open "Longhorn - Node" dashboard | | Node metrics dashboard displays               |
+| 8 | Verify volume health metrics display correctly | | Volume status, replica count, and health shown |
+| 9 | Verify storage utilization metrics display | | Storage usage percentages and capacity shown  |
+| 10 | Verify performance metrics (IOPS, throughput) display | | Performance data visible and updating         |
+| 11 | Create test volume and verify new metrics appear | | New volume metrics reflected in dashboard     |
+
+### Alert Manager Integration
+
+Test Longhorn alert integration with Rancher Alert Manager for storage-related events and notifications.
+
+| Step Number | Action | Data | Expected Result                           |
+|------|--------|--------|--------------------------------------------|
+| 1 | Verify Rancher monitoring and alerting are configured | | Alert Manager operational                  |
+| 2 | Navigate to "Monitoring" → "Alertmanager" in Rancher | | Alert Manager interface accessible         |
+| 3 | Verify Longhorn alert rules exist in Prometheus rules | | Rules for volume/node alerts present       |
+| 4 | Configure notification channel (Slack/email) for testing | | Notification channel configured            |
+| 5 | Simulate volume degraded state by shutting down node | | Node shutdown triggers volume degradation  |
+| 6 | Verify alert appears in Alert Manager within 2 minutes | | `LonghornVolumeStatusCritical` alert fires |
+| 7 | Check notification delivery through configured channel | | Alert notification received                |
+| 8 | Simulate storage space warning by filling disk | | Disk usage exceeds threshold               |
+| 9 | Verify storage warning alert triggers | | `LonghornDiskStorageWarning` alert fires   |
+| 10 | Resolve issues and verify alert resolution | | Alerts automatically resolve               |
+| 11 | Confirm resolution notifications sent | | Resolution notifications received          |
+
+### Backup Target Configuration
+
+Test backup target setup and configuration for S3 and NFS storage through Rancher interface.
```

Comment:
I would create one test for S3 and another for NFS if we want testing for both.

Reply:
Taking out NFS